tests/smoke_test/test_smoke_synapse_connectivity.py

```diff
@@ -17,5 +17,9 @@ def test_synapse_connectivity(self):
 
         notebook_run_result = self.run_notebook(notebook_name, notebook_parameters)
         exit_value = notebook_run_result["result"]["exitValue"]
-        failing_tests = json.loads(exit_value)
-        assert not failing_tests, f"The following tests failed in the test_smoke_py_connectivity notebook : {json.dumps(failing_tests, indent=4)}"
+        if exit_value:
+            exit_value_cleaned = ast.literal_eval(exit_value)
```

Comment:
Using literal_eval it can come back as string ? Isn't it better to use JSON end to end ? do a json.dumps and the test do a json.loads(exit_value) ? so you don't have surprises with the eval ?

Reply:
Good idea, i'll add this in